PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -254,7 +254,7 @@ void FemtoDreamTrackSelection::init(HistogramRegistry* registry, const std::stri
     }
 
     /// check whether the number of selection exceeds the bitmap size
-    int nSelections = getNSelections() - getNSelections(femtoDreamTrackSelection::kPIDnSigmaMax);
+    size_t nSelections = getNSelections() - getNSelections(femtoDreamTrackSelection::kPIDnSigmaMax);
```

Comment:
Remember that you can always use `unsigned int` when you are comparing with sizes